core/management/commands/data_intake.py

```diff
@@ -47,12 +51,16 @@ def handle(self, *args, **kwargs):
                 age=row["age"],
                 nhs_number=row["nhs_number"],
             )
-            datasource = DataSource.objects.create(
-                name=filename,
-                last_update=timezone.now(),
-            )
             record = Record.objects.create(
                 person_id=person,
                 datasource_id=datasource,
                 record=json_records[ind],
             )
+
+    def handle(self, *args, **kwargs):
+        Record.objects.all().delete()
+        DataSource.objects.all().delete()
+        Person.objects.all().delete()
+
```

Comment:
Clearing the db each time while we test. 
This part will be removed when there is actual data that shouldn't be deleted.

Reply:
Good idea, but would it be better to have this as a separate command instead of in the data intake?  Happy to leave it for now.